### PR TITLE
feat: expand distance results

### PR DESCRIPTION
**Pull request by notrab**
```md
### Understanding Distance Results

The `vector_distance_cos` function calculates the cosine distance, which equals 1 - [cosine
similarity](https://en.wikipedia.org/wiki/Cosine_similarity). Therefore:

- A distance close to 0 (including very small negative numbers like `-6.394e-14`) indicates that the vectors are nearly identical or exactly matching
- Values closer to 1 indicate that the vectors are dissimilar
- Typical distance values for non-matching vectors often appear as decimal numbers between 0 and 1

OR

<Note>
  Note: Very small negative numbers close to zero (on the order of `10^-14`) are
  due to floating-point arithmetic precision and should be interpreted as
  effectively zero, indicating an exact or near-exact match between vectors.
</Note>
```

### CODE REVIEW

File: features/ai-and-embeddings.mdx [hunk 1]
```diff
@@ -27,30 +27,39 @@ Full support for vector search in the Turso platform starts from version `v0.24.
 LibSQL uses the native SQLite BLOB [storage class](https://www.sqlite.org/datatype3.html#storage_classes_and_datatypes) for vector columns. To align with SQLite [affinity rules](https://www.sqlite.org/datatype3.html#determination_of_column_affinity), all type names have two alternatives: one that is easy to type and another with a `_BLOB` suffix that is consistent with affinity rules.
 
 <Info>
-We suggest library authors use type names with the `_BLOB` suffix to make results more generic and universal. For regular applications, developers can choose either alternative, as the type name only serves as a **hint** for SQLite and external extensions.
+  We suggest library authors use type names with the `_BLOB` suffix to make
```
Comment (sivukhin) [inline features/ai-and-embeddings.mdx:30]: Why to break line here?

File: features/ai-and-embeddings.mdx [hunk 2]
```diff
@@ -115,22 +122,30 @@ SELECT title,
...
 
 </Steps>
 
+### Understanding Distance Results
+
+The `vector_distance_cos` function calculates the cosine distance, which equals 1 - [cosine
+similarity](https://en.wikipedia.org/wiki/Cosine_similarity). Therefore:
+
+- A distance close to 0 (including very small negative numbers like `-6.394e-14`) indicates that the vectors are nearly identical or exactly matching
+- Values closer to 1 indicate that the vectors are dissimilar
```
Comment (sivukhin) [inline features/ai-and-embeddings.mdx:140]: cosine distance actually lies in range `[0..2]` - so values closer to 2 indicate that vectors are opposite, and values close to 1 indicate that vectors are orthogonal

File: features/ai-and-embeddings.mdx (hunk 1 above)
Comment (notrab) [reply features/ai-and-embeddings.mdx:30]: Prettier

**Review by sivukhin — APPROVED** [review]
